submissions/damilolaSalami/html/productpage.html

```diff
@@ -11,10 +11,10 @@
         <h1><a href="#">BuyMe</a></h1>
         <nav>
```

Comment:
The changes made on this file revert it. It will conflict with the other pull request. You are only suppose to create the contact form. @queensaldam